lib/gooddata/lcm/actions/synchronize_users.rb

```diff
@@ -317,11 +236,88 @@ def call(params)
           errors = results.select { |r| r[:type] == :error || r[:type] == :failed }
           return if errors.empty?
 
-          puts "Printing 10 first errors"
-          puts "========================"
+          puts 'Printing 10 first errors'
+          puts '========================'
           pp errors.take(10)
           fail 'There was an error syncing users'
         end
+
+        def load_data(params, data_source)
+          first_name_column           = params.first_name_column || 'first_name'
+          last_name_column            = params.last_name_column || 'last_name'
+          login_column                = params.login_column || 'login'
+          password_column             = params.password_column || 'password'
+          email_column                = params.email_column || 'email'
+          role_column                 = params.role_column || 'role'
+          sso_provider_column         = params.sso_provider_column || 'sso_provider'
+          authentication_modes_column = params.authentication_modes_column || 'authentication_modes'
+          user_groups_column          = params.user_groups_column || 'user_groups'
+          language_column             = params.language_column || 'language'
+          company_column              = params.company_column || 'company'
+          position_column             = params.position_column || 'position'
+          country_column              = params.country_column || 'country'
+          phone_column                = params.phone_column || 'phone'
+          ip_whitelist_column         = params.ip_whitelist_column || 'ip_whitelist'
+          mode                        = params.sync_mode
+
+          sso_provider = params.sso_provider
+          authentication_modes = params.authentication_modes || []
+
+          multiple_projects_column = params.multiple_projects_column
+          unless multiple_projects_column
+            client_modes = %w[sync_domain_client_workspaces sync_one_project_based_on_custom_id sync_multiple_projects_based_on_custom_id]
```

Comment:
Style/PercentLiteralDelimiters: `%w`-literals should be delimited by `(` and `)`.